pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/RuntimeASTTransformer.groovy

```diff
@@ -77,6 +77,16 @@ class RuntimeASTTransformer {
             false
     )
 
+    /**
+     * Enables or disables allowing local variable declarations while script splitting.
+     * This severely reduces the effectiveness of script splitting.
+     */
+    @SuppressFBWarnings(value="MS_SHOULD_BE_FINAL", justification="For access from script console")
+    public static boolean SCRIPT_SPLITTING_ALLOW_LOCAL_VARIABLES = SystemProperties.getBoolean(
+            RuntimeASTTransformer.class.getName() + ". SCRIPT_SPLITTING_ALLOW_LOCAL_VARIABLES",
```

Comment:
Line 86 contains a space before `SCRIPT_SPLITTING_ALLOW_LOCAL_VARIABLES`.

Reply:
@Benjamindv
Thank you for finding this. Perhaps next time open a PR.  Commenting on closed PRs and issues is a great way for your comments to not be seen.